lib/ActivityStreams.js

```diff
@@ -443,6 +470,21 @@ ActivityStreams.prototype = {
   },
 
   /**
+    * Start a timer to fetch a new recommendation every hour. This will only
+    * run for those in the experiment
+    */
+  _refreshRecommendations(recommendationTTL) {
+    if (recommendationTTL) {
+      this._newRecommendationTimeoutID = setTimeout(() => {
```

Comment:
That means the timeout will run forever for everyone, regardless of whether the pref is on